tests/api/test_acl_counter.py

```diff
@@ -1,6 +1,9 @@
 from pprint import pprint
 
+import pytest
 
+
+@pytest.mark.npu
```

Comment:
To avoid the warnings during execution, should be defined in https://github.com/opencomputeproject/SAI-Challenger/blob/main/tests/pytest.ini

Reply:
created  https://github.com/opencomputeproject/SAI-Challenger/pull/216 for this